packages/commonwealth/client/scripts/views/components/header/login_selector.tsx

```diff
@@ -511,7 +517,12 @@ export const LoginSelector = () => {
             <CWButton
               buttonType="tertiary-black"
               onClick={async () => {
-                if (sameBaseAddressesRemoveDuplicates.length === 0) {
+                if (
+                  sameBaseAddressesRemoveDuplicates.length === 0 ||
+                  app.chain?.meta?.id === 'injective' ||
+                  (app.user.addresses[0].address.slice(0, 3) === 'inj' &&
```

Comment:
Can you explain this line? Is the 0 index address the same as active?

Reply:
Updated to use active address